src/adapter/etl-adapter-google-sheet/src/Flow/ETL/Adapter/GoogleSheet/GoogleSheetExtractor.php

```diff
@@ -77,14 +77,14 @@ function (array $rowData) use ($headers, $shouldPutInputIntoRows, &$totalRows) {
                         /** @var int $totalRows */
                         $totalRows++;
 
+                        $row = \array_combine($headers, $rowData);
+
                         if ($shouldPutInputIntoRows) {
-                            return \array_merge(
-                                \array_combine($headers, $rowData),
-                                ['spread_sheet_id' =>  $this->spreadsheetId, 'sheet_name' => $this->columnRange->sheetName]
-                            );
+                            $row['spread_sheet_id'] = $this->spreadsheetId;
+                            $row['sheet_name'] = $this->columnRange->sheetName;
```

Comment:
This should be prefixed with `_`, I missed that while I was working on the extractors, could you please adjust it?

Reply:
Adjusted.